packages/neuvector/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,58 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -4,8 +4,8 @@
+ 
+ openshift: false
+ 
+-registry: registry.neuvector.com
+-tag: 4.4.3
++registry: docker.io
++tag: 5.0.0-preview.1
+ oem:
+ imagePullSecrets:
+ psp: false
+@@ -20,7 +20,7 @@
+       maxSurge: 1
+       maxUnavailable: 0
+   image:
+-    repository: neuvector/controller
++    repository: neuvector/controller.preview
```

Comment:
All of these need to be in format of image: repository/tag
https://github.com/rancher/system-charts/blob/cb84791246fad83dd4bfe909fc7245ea23f69fa1/charts/rancher-monitoring/v0.1.3/charts/prometheus/templates/prometheus.yaml#L26

Reply:
In the value.yaml, we intentionally make the tag and repo in separate values. This is to prevent users from giving different release versions to controller/enforcer/manager accidentally.